App/Properties.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace App
+{
+    public class Properties
```

Comment:
Should probably give credit to the person who wrote this, also documentation would be nice